src/api-detection/mobile-libraries-loader.provider.js

```diff
@@ -4,34 +4,27 @@
     angular.module('AppDetection')
         .provider('MobileLibrariesLoader', MobileLibrariesLoaderProvider);
 
-    var defaults = [
-        'bower_components/angular-touch/angular-touch.js',
-        'bower_components/angular-animate/angular-animate.js',
-        'bower_components/angular-route/angular-route.js',
-        'angular-jqm.js',
-    ];
-
     /**
      * @ngdoc provider
      * @name AppDetection.provider:MobileLibrariesLoader
      * @description
      * Loads libraries targeted at mobile devices
      */
-    function MobileLibrariesLoaderProvider() {
+    function MobileLibrariesLoaderProvider(BaseUrlSetterProvider, PROJECT_DATA, DETECTION_CONFIG) {
```

Comment:
Loading an extra library is slow on mobile. I would remove add angular-touch as an optional module like ui-router.
MobileLibrariesLoaderProvider can be removed also.